pkg/cmd/common.go

```diff
@@ -60,6 +66,14 @@ func addQueryBackendOptionsFlags(cmd *cobra.Command, options *query.QueryBackend
 	cmd.Flags().BoolVar(&options.UseProxy, "use-proxy", false, "Instead of temporarily port-forwarding, proxy a request to Kubecost through the Kubernetes API server.")
 	cmd.Flags().StringVarP(&options.KubecostNamespace, "kubecost-namespace", "N", "kubecost", "The namespace that kubecost is deployed in. Requests to the API will be directed to this namespace.")
 	cmd.Flags().StringVar(&options.AllocationPath, "allocation-path", "/model/allocation", "URL path at which Allocation queries can be served from the configured service. If using OpenCost, you may want to set this to '/allocation/compute'")
+
+	//Check if environment variable KUBECTL_COST_USE_PROXY is set, it defaults to false
+	v := viper.New()
+	v.SetEnvPrefix(envPrefix)
+	v.AutomaticEnv()
+	options.UseProxy = v.GetBool("KUBECTL_COST_USE_PROXY")
```

Comment:
If we're setting an env prefix, do we really need `KUBECTL_COST_USE_PROXY` here, or can we just use `USE_PROXY`?

Reply:
Actually this was a harmless cruft from my previous misunderstood removal of --use-proxy flag. I removed it.